### ISSUE TITLE
Russian capital letter "И" instead of Belarusian "І"

### ISSUE THREAD

**Issue by wiktorkowalczuk** [auseika/macos-by-keyboard#3]
Way to reproduce:

1. Install the keyboard
2. Initiate Caps Lock
3. Type "І" in any text editor

Expected result: "І" appears
Actual result: "И" appears

**Comment by auseika** [comment]
@wiktorkowalczuk yep, it seems like this is the issue with PC keyboard only, and everything works fine while using shift instead of caps lock. Will fix today.

**Comment by auseika** [comment]
@wiktorkowalczuk should be fixed now, reboot or logout after updating
https://github.com/auseika/macos-by-keyboard/releases/tag/v.4
